html/partials/compare/repositories.html.haml

```diff
@@ -18,6 +18,11 @@
             - if @diff["repositories"].only_in2
               %span.summary-part
                 #{@description_b.name}: #{@diff["repositories"].only_in2.length} repositories
+            - if @diff["repositories"].changed
+              %span.summary-part
+                %a{ href: "#repositories_changed" }
+                  changed
+                = ": #{@diff["repositories"].changed.length} repositories"
```

Comment:
The text has the wrong form, when only one repository is changed. Then it says "1 repositories", but it should say "1 repository"